arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -1696,13 +1696,13 @@
     },
     "LoadParameters": {
       "type": "object",
-      "description": "Parameters required for endpoint load.",
+      "description": "Parameters required for content load.",
       "required": [
         "contentPaths"
       ],
       "properties": {
         "contentPaths": {
-          "description": "The path to the content to be loaded. Should describe a file path.",
+          "description": "The path to the content to be loaded. Path should be a fule URL.",
```

Comment:
This is still not clear. Should it be a relative or an absolute file path or it can be any of the two? Can the description be clear on this.
`Path should be a relative or an absolute file path."` URL is more confusing. when you say url the user may think you are talking about "file://foo/bar" a url starting with a file protocol. I think that is not the case here, correct?